stock_quantity_history_location/wizards/stock_quantity_history.py

```diff
@@ -13,7 +13,7 @@ class StockQuantityHistory(models.TransientModel):
     location_id = fields.Many2one(
         "stock.location", domain=[("usage", "in", ["internal", "transit"])]
     )
-    include_child_locations = fields.Boolean("Include child locations", default=True)
+    include_child_locations = fields.Boolean("Include Child locations", default=True)
```

Comment:
IMHO, you can remove the string arg as it will be automatically generated from field name.

Reply:
Is there anything missing to upgrade?